src/components/Tooltip/Tooltip.slots.tsx

```diff
@@ -0,0 +1,8 @@
+import { FC, ReactElement } from 'react';
+
+interface TooltipSlotProps {
+	children: ReactElement;
+}
+
+export const TooltipContent: FC<TooltipSlotProps> = ({ children }) => children;
+export const TooltipTrigger: FC<TooltipSlotProps> = ({ children }) => children;
```

Comment:
everywhere else in the code we use `FunctionComponent` instead of `FC`
better to keep it uniform?